api/android/README.md

```diff
@@ -235,3 +235,44 @@ You can find the result in ```$NNSTREAMER_ROOT/android_lib```.
 $ cd $NNSTREAMER_ROOT
 $ bash ./api/android/build-android-lib.sh --run_test=yes
 ```
+
+### Using TensorFlow Lite NNAPI Delegate
+
+Since Android 10, all applications are in scoped storage and the `getExternalStorageDirectory()` is deprecated.
+There are some workarounds to use files with external path (e.g. set `requestLegacyExternalStorage` true in app's manifest file). With TensorFlow Lite model files in external storage, however, all workarounds do not work when an application attempts to access the NNAPI, and TensorFlow Lite fails to use available backend like GPU, DSP, or NPU.
+
+If you want to use NNAPI delegate, you should provide the model file from internal storage. You may copy the model files into app-specific directory or cache directory. Or let your application access model files reside in the asset directory.
+
+Here are sample java methods:
+
+```java
+/*
+ * Copy a model file in external storage into app-specific internal storage and return it
+ *
+ * @param context  The application context
+ * @param model    The File object of a model file in external storage
+ */
+File modelFromFilesDir(Context context, File model) {
+  File appSpecificFile = new File(context.getFilesDir(), "model_file_in_FilesDir.tflite");
```

Comment:
new File(context.getFilesDir(), model.getName()); maybe this is right one

Reply:
Yeah that would be great! I updated this.